department-of-veterans-affairs/caseflow
# Craig/appeals 26096

Resolves [APPEALS-26096](https://jira.devops.va.gov/browse/APPEALS-26096)

# Description
- Modify VHA seed file to create HLRs and SCs with claimants (and powers of attorney) and tasks to more accurately model a case that has gone through intake
- Modify VHA seed file to create 10182 Appeals with the correct task tree at VHA CAMO/Caregiver, program offices, and regional offices in each tab of their queue
- Removed duplicate/confusing tasks from the task factory
- Added a trait to HLR/SC factories to create a request issue based on benefit type of the decision
- Added logic to the `after(:create)` hook in the HLR/SC factories to create a claimant based on parameters passed in

## Acceptance Criteria
- [ ] Cases exist in each VISN, PO, VHA CAMO, VHA Caregiver queue tab
- [ ] HLRs and SCs in the decision review queue can be viewed and have POA data associated if applicable (may require redux to view until feature work is completed)

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Verify that cases exist in each tab for VHA Camo, VHA Caregiver, a program office, and a regional office
3. Verify that HLRs and SCs have POA data (may need to be done with redux or rails console)

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
